### PR TITLE
address issue where empty docker auth credentials are used to create …

### PR DESCRIPTION
**Pull request by paselem**
…the cluster

#189 
#188

### CODE REVIEW

File: R/cluster.R [hunk 1]
```diff
@@ -156,13 +157,15 @@ makeCluster <-
 
     # log into private registry if registry credentials were provided
     if (!is.null(config$dockerAuthentication) &&
-        !is.null(config$dockerAuthentication$username)) {
+        !is.null(config$dockerAuthentication$username) &&
+        nchar(config$dockerAuthentication$username) > 0 &&
+        nchar(config$dockerAuthentication$password) > 0 &&
```
Comment (zfengms) [inline R/cluster.R]: We have is.null check for username, do we need is.null on password and registry as well?

Comment (brnleehng) [inline R/cluster.R]: You can remove the !is.null(config$dockerAuthentication$username) since nchar(NULL) > 0 will cover that case for you

Comment (paselem) [reply R/cluster.R]: I think so because they could in theory fill in any one of the three values and it will fail unless they are all filled out.

Comment (paselem) [reply R/cluster.R]: Done.

**Pull request merged**